CMakeLists.txt

```diff
@@ -57,7 +57,12 @@ project(${BOARD_LIB_NAME}
         LANGUAGES CXX C
         )
 
-add_library(${PROJECT_NAME} STATIC)
+add_library(${PROJECT_NAME} STATIC
+        include/LVSS.hpp
+        src/dev/TPS2HB50BQ1.cpp
+        include/dev/TPS2HB50BQ1.hpp
+        src/dev/ACS781XLR.cpp
+        include/dev/ACS781XLR.hpp)
```

Comment:
Pretty sure you don't need to change this. Just add the source files to target_sources